lib/constants.js

```diff
@@ -24,19 +24,18 @@ module.exports = {
     FREE_THRESHOLD: 'free-threshold'
   },
   PROMO_EXPIRES: {
-    'DEFAULT': setPromoExpiration(1, 'year'),
-    'NEW_SIGNUP': setPromoExpiration(3, 'months'),
-    'REFERRAL_RECIPIENT': setPromoExpiration(3, 'months'),
-    'REFERRAL_SENDER': setPromoExpiration(3, 'months'),
+    DEFAULT: setPromoExpiration(12, 'months'),
+    NEW_SIGNUP: setPromoExpiration(12, 'months'),
+    REFERRAL_RECIPIENT: setPromoExpiration(3, 'months'),
+    REFERRAL_SENDER: setPromoExpiration(3, 'months'),
     FREE_THRESHOLD: new Error('FREE_THRESHOLD expiration should be' +
       'calculated to be one month after the billing date in the current month')
   },
-  // NB: TODO: THESE VALUES SHOULD ALL BE IN TERMS OF CENTS!!!
   PROMO_AMOUNT: {
-    'NEW_SIGNUP': 4.88,
-    'REFERRAL_RECIPIENT': 9.75,
-    'REFERRAL_SENDER': 9.75,
-    'MIN_SPENT_REQUIREMENT': 10,
+    NEW_SIGNUP: 488,
+    REFERRAL_RECIPIENT: 1000,
```

Comment:
We bumped it to $10 even. It allows us to be flexible with how we promote referrals as well. It's either approx. 50GB/50GB every month for three months, or it's a $10 credit valid for 3 months, use how you will.